.travis.yml

```diff
@@ -16,6 +16,7 @@ script:
   # instruct bash to quit if there is a non zero response code (error)
   # https://github.com/commercetools/ui-kit/pull/284
   - set -e
+  - yarn run jest --projects jest.eslint.config.js --maxWorkers=4 --reporters jest-silent-reporter
   - yarn run jest --projects jest.test.config.js --maxWorkers=4 --reporters jest-silent-reporter
```

Comment:
I think we can 

```
yarn run jest --projects jest.*.config.js --maxWorkers=4 --reporters jest-silent-reporter
```

Reply:
no, because we need to exclude `jest.bundle.config` and `jest.visual.config` from the run until the bundle is built.

Reply:
but maybe we can do 
```
yarn run jest --projects jest.eslint.config.js jest.test.config.js --maxWorkers=4 --reporters jest-silent-reporter
```

Reply:
fixed in [d022a222002535f2e3d8a0a062c527a9245c9436](https://github.com/commercetools/ui-kit/pull/434/commits/d022a222002535f2e3d8a0a062c527a9245c9436)

Reply:
or even

```
yarn run jest --projects jest.{eslint,test}.config.js --maxWorkers=4 --reporters jest-silent-reporter
```

doesn't really matter though.